src/ParametricUtils.jl

```diff
@@ -102,13 +102,10 @@ end
 
 # This is where the actual parametric calculation happens, CalcFactor equivalent for parametric
 function (cfp::CalcFactorMahalanobis)(variables...)
-  #TODO res might be removed see https://github.com/JuliaRobotics/IncrementalInference.jl/issues/467#issuecomment-772987921
-  res = Vector{eltype(variables[1])}(undef, length(cfp.meas))
-  iΣ = cfp.iΣ
-  cfp.calcfactor!(res, cfp.meas, variables...)
-  
+  # call the user function (be careful to call the new CalcFactor version only!!!)
+  res = cfp.calcfactor!(cfp.meas, variables...)
```

Comment:
Also expand measurements if more than one (`getSample` always returns a `::Tuple`):
```julia
res = cfp.calcfactor!(cfp.meas..., variables...)
```

Reply:
This is the measurement from `getParametricMeasurement` it currently returns a `Vector{Float64}`. In what cases would a parametric solve be able to handle more than this? An example or use case would be helpful for development and testing

Reply:
okay, lets leave as you have it for now.